src/super_gradients/training/utils/segmentation_utils.py

```diff
@@ -21,6 +26,19 @@ def __repr__(self):
         return self.__class__.__name__ + str(self.__dict__).replace('{', '(').replace('}', ')')
 
 
+class ResizeSeg(SegmentationTransform):
```

Comment:
@shaydeci  - We need to create a proper place in the modules hierarchy for all of the Augmentations instead of putting everything inside utils...